models/hospital.go

```diff
@@ -0,0 +1,7 @@
+package models
+
+type Hospital struct {
+	Id int `db:"id" json:"id"`
+	Name string `db:"name" json:"name"`
+	MaxPatientAmount int `db:"max_patient_amount" json:"max_patient_amount"`
```

Comment:
The json representation should be `camelCase`. E.g. 
```suggestion
	MaxPatientAmount int `db:"max_patient_amount" json:"maxPatientAmount"`
```
Do this for every model's attributes.